twopane-navigation/src/twoPane/twoPaneStore/tests/twoPane.methods.test.tsx

```diff
@@ -0,0 +1,256 @@
+import React from 'react'
+import twoPane from '../twoPane.methods';
+import * as keyActions from '../../../shared/screenStore/keyStore/key.actions';
+import * as headerActions from '../../../shared/screenStore/headerStore/header.actions';
+import * as paneElementActions from '../../../shared/screenStore/paneElementStore/paneElement.action';
```

Comment:
Love to see the tests!  How do you run them?  Also we should setup the CI loop to run them (can be tracked as a separate issue).

Reply:
sure you just have to run npm run test. to give them a spin currently the code coverage is averaging around 80%. personally i would love to shoot for upper 90's but i figured one thing at a time.